docs/process/integrating-with-fxa.md

```diff
@@ -4,28 +4,123 @@ title: Integration with FxA
 
-### Sample Timeline
+The meeting request should be sent to fxa-staff[at]mozilla.com. Please be prepared to discuss these integration topics with the FxA team.
 
-Turnip greens yarrow ricebean rutabaga endive cauliflower sea lettuce kohlrabi amaranth water spinach avocado daikon napa cabbage asparagus winter purslane kale. Celery potato scallion desert raisin horseradish spinach carrot soko. Lotus root water spinach fennel kombu maize bamboo shoot green bean swiss chard seakale pumpkin onion chickpea gram corn pea. Brussels sprout coriander water chestnut gourd swiss chard wakame kohlrabi beetroot carrot watercress. Corn amaranth salsify bunya nuts nori azuki bean chickweed potato bell pepper artichoke.
+## OAuth Integration
 
-## For Product Managers
+### Development
 
-Turnip greens yarrow ricebean rutabaga endive cauliflower sea lettuce kohlrabi amaranth water spinach avocado daikon napa cabbage asparagus winter purslane kale. Celery potato scallion desert raisin horseradish spinach carrot soko. Lotus root water spinach fennel kombu maize bamboo shoot green bean swiss chard seakale pumpkin onion chickpea gram corn pea. Brussels sprout coriander water chestnut gourd swiss chard wakame kohlrabi beetroot carrot watercress. Corn amaranth salsify bunya nuts nori azuki bean chick
+1. A basic understanding of [OAuth 2.0][oauth] is required.
+2. Register for OAuth credentials at https://oauth-stable.dev.lcip.org/console/login. See [OAuth credentials](#oauth-credentials).
+3. Development servers point to: https://stable.dev.lcip.org
+4. User authentication follows the [OAuth 2.0][oauth] protocol.
+5. If the multiple Service Providers (SPs) are accessed, distinct tokens are created with appropriate scopes for each SP.
+6. [Query parameters](#authorization-query-parameters) are set and validate when redirecting to Firefox Accounts.
+7. [Self hosted email-first](#self-hosted-email-first-flow) flows initialize and propagate top of funnel metrics.
+8. [User data and account notifications are properly](#user-data-hygiene-gdpr) handled and GDPR compliant.
+9. An icon suitable to display in FxA’s [Devices & apps][devices-and-apps] list has been sent to FxA devs.
 
-## For Engineers
+### Preparing for Production
 
-Turnip greens yarrow ricebean rutabaga endive cauliflower sea lettuce kohlrabi amaranth water spinach avocado daikon napa cabbage asparagus winter purslane kale. Celery potato scallion desert raisin horseradish spinach carrot soko. Lotus root water spinach fennel kombu maize bamboo shoot green bean swiss chard seakale pumpkin onion chickpea gram corn pea. Brussels sprout coriander water chestnut gourd swiss chard wakame kohlrabi beetroot carrot watercress. Corn amaranth salsify bunya nuts nori azuki bean chick
+1. Register for a production OAuth credentials by filing a [deployment bug][deployment-bug]. See [OAuth credentials](#oauth-credentials).
+2. Production servers point to with https://oauth.accounts.firefox.com.
```

Comment:
sp: "point to with"?

Also do we still want them to point to this domain, or should it be the new locations under https://auth.accounts.firefox.com, or perhaps values discovered dynamically from https://accounts.firefox.com/.well-known/openid-configuration.